src/katgpucbf/xbgpu/xsend.py

```diff
@@ -119,7 +119,7 @@ def timestamp(self) -> int:  # noqa: D102
         return int(self._timestamp[()])
 
     @timestamp.setter
-    def timestamp(self, value: int) -> None:  # noqa: D102
+    def timestamp(self, value: int) -> None:
```

Comment:
I see the `noqa` moniker is still on the `@property` above.
* Does it now hold for all subsequent/related usages of the property?
* i.e. Is this `noqa` removed because it's mentioned elsewhere?

Reply:
Yes ruff seems to be smarter about ignoring property setters.